server/connection.c

```diff
@@ -66,6 +70,13 @@ AP_DECLARE(conn_rec *) ap_create_connection(apr_pool_t *p,
     return c;
 }
 
+AP_DECLARE(conn_rec *) ap_create_secondary_connection(apr_pool_t *p,
+                                                      conn_rec *master,
+                                                      apr_bucket_alloc_t *alloc)
+{
+    return ap_run_create_secondary_connection(p, master, alloc);
+}
```

Comment:
I'm not sure we need a new `ap_create_secondary_connection()` for now if it's only a pass through `ap_run_create_secondary_connection()`

Reply:
I think it is good to have and at no cost. We have added wrappers around hooks in the past, when things turned out to need global checks or additional things.

Reply:
Yeah right but if at that time for instance we need more parameters in the helper than in the hook we'd have to create and _ex() version if the helper exists already.
Not a strong opinion though..

Reply:
Not opposed to it either. Maybe @rpluem can arbitrate...